src/unstract/sdk/llm.py

```diff
@@ -71,15 +72,15 @@ def run_completion(
                 ].llm_token_counts
                 if llm_token_counts:
                     llm_token_count = llm_token_counts[0]
-                    usage[
-                        "prompt_token_count"
-                    ] = llm_token_count.prompt_token_count
-                    usage[
-                        "completion_token_count"
-                    ] = llm_token_count.completion_token_count
-                    usage[
-                        "total_token_count"
-                    ] = llm_token_count.total_token_count
+                    usage["prompt_token_count"] = (
```

Comment:
NIT: I know this is a formatting change however consider defining and using constants here